cmd/bmp180/main.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2016 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+// bmp180 reads the current temperature and pressure from a BMP180.
```

Comment:
Uhm, that's not package `bmp180`, but package `main`. The package `bmp180` has a proper doc string and a link to the datasheet:
```
// Package bmp180 controls a Bosch BMP180 device over I²C.
//
// Datasheet
//
// https://cdn-shop.adafruit.com/datasheets/BST-BMP180-DS000-09.pdf
```

The datasheet you linked is really bad btw, the font in the algorithm on page 15 is unreadable. The one at adafruit that I linked has at least a readable font in the screenshot...